components/patterns/navs/nav-mobile.stories.js

```diff
@@ -0,0 +1,39 @@
+import { withA11y } from '@storybook/addon-a11y'
+
+import {
+  IconMenuOutline,
+  IconNotificationsActive,
+  IconNotifications,
+  MtlNavMobile,
+  MtlSideBarMobile
+} from '@/components/index'
+export const NavMobile = () => ({
+  components: {
+    IconMenuOutline,
+    IconNotificationsActive,
+    IconNotifications,
+    MtlNavMobile,
+    MtlSideBarMobile
+  },
+  data: () => ({
+    open: false
+  }),
+  methods: {
+    toggleNav() {
+      // eslint-disable-next-line no-console
+      console.log('toggle nav')
```

Comment:
you should remove `console.log` and `eslint-disable` rule

Reply:
I have mixed feelings about this one, `stories` are for documentation and examples purposes, and `console.log` are a good way to show how things work.
I wouldn't mind leaving logs in stories.
What do you think?

Reply:
on this case what no using a `console.info()`? make more sense? if not I'm good with that.

Reply:
In storybook you like to have logs on actions. Normally it should be actions addon on storybook but i dont feel like its usefull for now. Yeah i can use info instead.